python/pysmurf/client/command/smurf_command.py

```diff
@@ -3390,6 +3390,7 @@ def flux_ramp_off(self, **kwargs):
         self.set_cfg_reg_ena_bit(0, **kwargs)
 
     _ramp_max_cnt = 'RampMaxCnt'
+    _ramp_max_cnt_clock_hz = 307.2e6
```

Comment:
Is this a thing we should hard-code into pysmurf? Can this ever change?

Reply:
@eyyoung24  good - yea, flagrant violation of my own beliefs.  See reason why in PR description ; hardcoding it gets around a rogue issue which otherwise makes it impossible to check if the requested flux ramp rate integer divides 307.2MHz.  Couldn't think of another way around.  I do think this is unlikely to change, but I also do hate hardcoded numbers.  We could drop reporting if the flux ramp set doesn't match the flux ramp requested if we want to compute instead.

Reply:
Actually, I think I just found a fix for this.  Will open a JIRA to try and understand why this happens, though.  Hold on merging @eyyoung24.

Reply:
Ok, resolving, let me know if doesn't look good to you.